packages/grafana-ui/src/components/BrowserLabel/Label.tsx

```diff
@@ -60,17 +77,22 @@ export const Label = forwardRef<HTMLElement, Props>(
         )}
         {...rest}
       >
-        <Highlighter
-          textToHighlight={text}
-          searchWords={searchWords}
-          autoEscape
-          highlightClassName={styles.matchHighLight}
-        />
+        {typeof highlightParts !== 'undefined' ? (
+          <PartialHighlighter text={text} highlightClassName={styles.matchHighLight} highlightParts={highlightParts} />
+        ) : (
+          <Highlighter
+            textToHighlight={text}
+            searchWords={searchWords}
+            autoEscape
+            highlightClassName={styles.matchHighLight}
+          />
+        )}
       </span>
     );
   }
 );
 
+// TODO: Connor - Change display name?
```

Comment:
I think this can stay as it is, it should match the component name. Not sure if we even still need this? @aocenas?

Reply:
I think the display name is normally derived from the named export so I'm not sure if we still need them. Before, the display names were Label and LokiLabel. I think we might be okay removing it.

Reply:
The displayName inference should work, unless you so something complicated.

Reply:
Cool, then I'll remove it.

Reply:
Looks like the display name is actually required (probably an eslint rule).